Cargo.toml

```diff
@@ -40,6 +40,9 @@ rust-version = "1.56"
 
 [features]
 alloc = [
+	# If the serde feature is also enabled, enable alloc for it
+	# so that we can deserialize Strings.
+	"serde?/alloc"
```

Comment:
Since the min supported rustc version is 1.56, this "?" features isn't available yet.
Removing it would mean that enabling the "alloc" features also enabled the "serde" feature. Any thoughts on how best to tackle this?

Reply:
In which version did this stabilize? While I had hoped to refrain from MSRV raises until I'm able to release 2.0 with the BitArray redesign, my MSRV policy *does* state that I may freely do so in the minor series.

If raising the MSRV to support optional transitive features really is sufficient (with the rest of this PR) to resolve the de/ser problem, then please raise MSRV in `rust-toolchain.toml` and `README.md`, and update to `1.1.0` in `Cargo.toml`.

I am going to continue working through my backlog in the `1.0` series. Thank you for your patience as I've ignored the project, and I look forward to releasing your 1.1 soon!